Assets/MRTK/SDK/StandardAssets/Models/Materials/Cheese2.mat

```diff
@@ -9,7 +9,8 @@ Material:
   m_PrefabAsset: {fileID: 0}
   m_Name: Cheese2
   m_Shader: {fileID: 4800000, guid: 5bdea20278144b11916d77503ba1467a, type: 3}
-  m_ShaderKeywords: _DIRECTIONAL_LIGHT _HOVER_LIGHT _REFLECTIONS _SPECULAR_HIGHLIGHTS
+  m_ShaderKeywords: _DIRECTIONAL_LIGHT _DISABLE_ALBEDO_MAP _HOVER_LIGHT _REFLECTIONS
```

Comment:
Just want to make sure these changes are intended.

Reply:
thanks,. no changes to the material were intended. i will fix that

Reply:
@MaxWang-MS fixed! nice catch